donormotor/templates/admin/donation_index.html

```diff
@@ -51,7 +51,12 @@ <h1>Donations</h1>
             <td>{{ donation.email }}</td>
             <td>{{ donation.phone }}</td>
             <td>{{ donation.placed_date|datetime }}</td>
-            <td>{% if donation.paid_date %}{{ donation.paid_date|datetime }}{% endif %}</td>
+	    <td>{% if donation.paid_date %}{{ donation.paid_date|datetime }}
```

Comment:
There may be an issue here with unpaid donations generated with cardtesting. These should not allow setting paid. c.f. #9

Reply:
also, whitespace needs fixed here.